portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+/**
+ * Adds a random fact.
+ */
+function addRandomFact() {
+    const facts = ["I have no sibling", "I seldom eat spicy food", "I don't like watching thrillers"];
```

Comment:
Maybe each string on one line since we have more than 2?

Reply:
Fixed